lib/fastlane/plugin/wpmreleasetoolkit/actions/common/create_release_backmerge_pull_request_action.rb

```diff
@@ -0,0 +1,213 @@
+                                  end
+
+                                  determine_target_branches(source_release_version: source_branch.delete('release/'), default_branch: default_branch)
+                                else
+                                  target_branches
+                                end
+
+        final_target_branches.map do |target_branch|
+          Fastlane::Helper::GitHelper.checkout_and_pull(source_branch)
+
+          create_backmerge_pr(
+            token: token,
+            repository: repository,
+            title: "Merge #{source_branch} into #{target_branch}",
+            head_branch: source_branch,
+            base_branch: target_branch,
+            labels: labels,
+            milestone: target_milestone&.number,
+            reviewers: reviewers,
+            team_reviewers: team_reviewers,
+            intermediate_branch_created_callback: intermediate_branch_created_callback
+          )
+        end.compact
+      end
+
+      # Determines the target branches for a release version.
+      #
+      # @param source_release_version [String] the source release version to compare against other release branches.
+      # @param default_branch [String] the default branch to use if no target branches are found.
+      # @return [Array<String>] the list of target branches greater than the release version.
+      def self.determine_target_branches(source_release_version:, default_branch:)
+        release_branches = Actions.sh('git', 'branch', '-r', '-l', 'origin/release/*').strip.split("\n")
+
+        all_release_branches_versions = release_branches
+                                        .map { |branch| branch.match(%r{origin/release/([0-9.]*)})&.captures&.first }
+                                        .compact
+
+        target_branches = all_release_branches_versions.select { |branch| Gem::Version.new(branch) > Gem::Version.new(source_release_version) }
+                                                       .map { |v| "release/#{v}" }
+        target_branches = [default_branch] if target_branches.empty?
+
+        target_branches
+      end
+
+      # Creates a backmerge pull request using the `create_pull_request` Fastlane Action.
+      #
+      # @param token [String] the GitHub token for authentication.
+      # @param repository [String] the repository where the pull request will be created.
+      # @param title [String] the title of the pull request.
+      # @param head_branch [String] the source branch for the pull request.
+      # @param base_branch [String] the target branch for the pull request.
+      # @param labels [Array<String>] the labels to add to the pull request.
+      # @param milestone [String] the milestone to associate with the pull request.
+      # @param reviewers [Array<String>] the individual reviewers for the pull request.
+      # @param team_reviewers [Array<String>] the team reviewers for the pull request.
+      # @return [void]
```

Comment:
```suggestion
      # @param team_reviewers [Array<String>] the team reviewers for the pull request.
      # @param intermediate_branch_created_callback [Proc?]
      #        A callback to call after having created the intermediate branch
      #        to allow the caller to e.g. push new commits on it before the PR is created
      # @return [String?] The URL of the created Pull Request, or `nil` if no PR was created.
```

Reply:
This made me realize that I think I've used `@param [Type] paramname description` in many places where I've written some YARD documentation, while indeed the expected format/order seems to be ``@param paramname [Type] description` instead (like you used here)… :oops:

Even if we don't really use the `yard` executable to generate HTML docs from our own code and in practice those comments are just for us when reading the code… I guess if we want to be consistent we should fix all the places (`Fastfile` in various repos, etc…) where we (I?) have used the wrong format 😅

Reply:
Updated on b4f411dfa291269c5b732a0c9704aa8b497f234a

> Even if we don't really use the yard executable to generate HTML docs from our own code and in practice those comments are just for us when reading the code… I guess if we want to be consistent we should fix all the places 

Yeah, since it's mostly internal, I guess we could change them as we move + do the new ones in the expected format 🙂